CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+## How to contribute
+
+Thank you for taking the time to contribute to the Helm chart for the 1Password SCIM bridge.
+
+#### **Did you find a bug?**
+
```

Comment:
Perhaps these should all be an ordered list, instead of unordered.

Reply:
I went back and forth on whether these lists should be ordered or unordered and ultimately kept it unordered for consistency.

Also, I quite like the way the Rails team did [theirs](https://github.com/rails/rails/blob/main/CONTRIBUTING.md), which I used as a starting point for this.